main.css

```diff
@@ -1,11 +1,15 @@
 header {
-    background-color: red;
```

Comment:
deleting background color and adding text color is a great decision! Visibility is improved!